tornado-drivers/drivers-common/src/main/java/uk/ac/manchester/tornado/drivers/common/compiler/phases/memalloc/TornadoNewArrayDevirtualizationReplacement.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, APT Group, Department of Computer Science,
+ * Copyright (c) 2023, APT Group, Department of Computer Science,
```

Comment:
```suggestion
 * Copyright (c) 2020, 2024 APT Group, Department of Computer Science,
```

Please, review all header files with the license to fix with the correct year.